lib/opentelemetry-proto/src/convert.rs

```diff
@@ -1,45 +1,47 @@
-use super::proto::{
-    common::v1::{any_value::Value as PBValue, KeyValue},
-    logs::v1::{LogRecord, ResourceLogs, SeverityNumber},
-    resource::v1::Resource,
-};
 use bytes::Bytes;
-use chrono::{TimeZone, Utc};
+use chrono::{DateTime, TimeZone, Utc};
+use lookup::path;
 use ordered_float::NotNan;
 use std::collections::BTreeMap;
 use value::Value;
 use vector_core::{
-    config::log_schema,
+    config::{log_schema, LegacyKey, LogNamespace},
     event::{Event, LogEvent},
 };
 
-const RESOURCE_KEY: &str = "resources";
-const ATTRIBUTES_KEY: &str = "attributes";
-const TRACE_ID_KEY: &str = "trace_id";
-const SPAN_ID_KEY: &str = "span_id";
-const SEVERITY_TEXT_KEY: &str = "severity_text";
-const SEVERITY_NUMBER_KEY: &str = "severity_number";
-const OBSERVED_TIMESTAMP_KEY: &str = "observed_timestamp";
-const DROPPED_ATTRIBUTES_COUNT_KEY: &str = "dropped_attributes_count";
-const FLAGS_KEY: &str = "flags";
+use super::proto::{
+    common::v1::{any_value::Value as PBValue, KeyValue},
+    logs::v1::{LogRecord, ResourceLogs, SeverityNumber},
+    resource::v1::Resource,
+};
+
+const SOURCE_NAME: &str = "opentelemetry";
```

Comment:
Seems like we should be using OpentelementryConfig::NAME instead of having this constant.

Reply:
I didn't want to have the library that Vector depends on, to depend on Vector itself. Once `otlp` becomes a decoder we could better reference that shared value.

Reply:
Oh missed that this was in /lib :+1: